CITATION.cff

```diff
@@ -0,0 +1,18 @@
+cff-version: 1.2.0
+
+type: conference-paper
+authors:
+  - family-names: Dreef
+    given-names: Kaj
+  - family-names: Palepu
+    given-names: Vijay Krishna
+  - family-names: Jones
+    given-names: James A.
+title: "Global Overviews of Granular Test Coverage with Matrix Visualizations"
+booktitle: "2021 Working Conference on Software Visualization (VISSOFT)"
```

Comment:
instead of things like book title, start, end ... (they are more focused on paper), shouldn't we have stuff like:
```
license: Apache-2.0
message: "If you use this software, please cite it using these metadata."
repository-code: "link to GitHub repo"
title: "Morpheus"
version: 0.x.y
```

feel free to change the values of license and version ... obviously.